ext/cache/01_cache.js

```diff
@@ -200,79 +205,80 @@
         requestUrl: r.url,
       });
     }
+  }
 
-    /** See https://w3c.github.io/ServiceWorker/#cache-matchall
-     *
-     * Note: the function is private as we don't want to expose
-     * this API to the public yet.
-     *
-     * The function will return an array of responses.
-     */
-    async #matchAll(request, _options) {
-      // Step 1.
-      let r = null;
-      // Step 2.
-      if (ObjectPrototypeIsPrototypeOf(RequestPrototype, request)) {
-        r = request;
-        if (request.method !== "GET") {
-          return [];
-        }
-      } else if (
-        typeof request === "string" ||
-        ObjectPrototypeIsPrototypeOf(URLPrototype, request)
-      ) {
-        r = new Request(request);
+  /** See https://w3c.github.io/ServiceWorker/#cache-matchall
+   *
+   * Note: the function is private as we don't want to expose
+   * this API to the public yet.
+   *
+   * The function will return an array of responses.
+   */
+  async function matchAll(request, _options, cacheId) {
```

Comment:
Removed private method since it fails after branding with: 
```
TypeError: Receiver must be an instance of class Cache
```

Reply:
you could use a symbol then isntead like we do for websocket: https://github.com/denoland/deno/blob/a3432e54c7214dd37b7318ad03442b8415fb4f67/ext/websocket/01_websocket.js#L393